### DISCUSSION TITLE
ScriptReference/Physics2D-alwaysShowColliders

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#630] [Notes]
# ScriptReference/Physics2D-alwaysShowColliders

https://docs.unity3d.com/ScriptReference/Physics2D-alwaysShowColliders.html

**Comment by qwe321qwe321qwe321** [comment]
No alternative solution to control gizmo from script as of today. (Unity 2022.3.21f1)
[Unity Forum Discussion](https://forum.unity.com/threads/set-physics-2d-gizmos-from-script.1439809/#post-9685598)
